### PR TITLE
Analysis software condor submission

### PR DESCRIPTION
**Pull request by kreczko**
Ready for merging

### CODE REVIEW

**Comment by kreczko** [comment]
redy for review

File: python/indices_cff.py [hunk 1]
```diff
@@ -107,8 +107,8 @@
 }
 muonRegions = {
     'signal': 'TopPairMuonPlusJetsSelection',
-    'controlRegion1': 'TopPairMuonPlusJetsQCDSelection1p5to3',
-    'controlRegion2': 'TopPairMuonPlusJetsQCDSelection3toInf',
+    'controlRegion1': 'TopPairMuonPlusJetsQCDSelection3toInf',
+    'controlRegion2': 'TopPairMuonPlusJetsQCDSelection1p5to3',
```
Comment (EmyrClement) [inline python/indices_cff.py]: Just checking these are now in the correct order.  Also in SelectionAnalyzer_cfi etc.

File: python/ntp/commands/run/local/analysis/__init__.py [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+PUFile_up = "{BAT_BASE}/data/" + settings['PUFile_up']
+PUFile_down = "{BAT_BASE}/data/" + settings['PUFile_down']
+getMuonScaleFactorsFromFile = True
+getElectronScaleFactorsFromFile = True
+ElectronIdScaleFactorsFile = '{BAT_BASE}/data/ElectronCutBasedID_MediumWP_76X_SF2D.root'
+ElectronIsoScaleFactorsFile = '{BAT_BASE}/data/Elec_SF_Fit_Syst.root'
+ElectronTriggerScaleFactorsFile = '{BAT_BASE}/data/ElectronTriggerEfficiencies.root'
+MuonIdScaleFactorsFile = '{BAT_BASE}/data/MuonID_Z_RunCD_Reco76X_Feb15.root'
+MuonIsoScaleFactorsFile = '{BAT_BASE}/data/MuonIso_Z_RunCD_Reco76X_Feb15.root'
+MuonTriggerScaleFactorsFile = '{BAT_BASE}/data/SingleMuonTrigger_Z_RunCD_Reco76X_Feb15.root'
```
Comment (EmyrClement) [inline python/ntp/commands/run/local/analysis/__init__.py]: Do we have to update the scale factor files here, in AT, or both now?

File: python/ntp/commands/run/local/analysis/__init__.py [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+
+#MET corrections application
+applyMetSysShiftCorr = False
+applyMetType0Corr = False
+
+#relative Path from calling BAT to the TopQuarkAnalysis folder
+TQAFPath = ""
+
+#integrated luminosity the MC simulation will be scaled to
+lumi = 15933
```
Comment (EmyrClement) [inline python/ntp/commands/run/local/analysis/__init__.py]: Same for lumi.  Does it have to be updated here, or in AT?

File: python/indices_cff.py (hunk 1 above)
Comment (kreczko) [inline python/indices_cff.py]: The definition can be found here: [MuonUserData.cc#L185](https://github.com/BristolTopGroup/NTupleProduction/blob/master/plugins/userdata/MuonUserData.cc#L185)

I agree, this is easy to mix up, maybe we should rename it?

Anyway, 
`TopPairMuonPlusJetsQCDSelection3toInf == muon.userInt("isGoodNonIsoR1") == TopPairMuonPlusJetsQCDSelection3toInf` etc:
 - [muonSelections_cff.py#L13](https://github.com/BristolTopGroup/NTupleProduction/blob/master/python/muonSelections_cff.py#L13)
 - [miniAODToNTuple_cfg.py#L332](https://github.com/BristolTopGroup/NTupleProduction/blob/master/python/run/miniAODToNTuple_cfg.py#L332)

File: python/ntp/commands/run/local/analysis/__init__.py (hunk 3 above)
Comment (kreczko) [inline python/ntp/commands/run/local/analysis/__init__.py]: Apologies, I thought I replied to this one.
If you are running AT with the ntp commands you would change it in `python/ntp/commands/run/local/analysis/__init__.py `

I have not found an easy way to import the config from AT, modify it and then write the new config file. A limitation on how we implemented the python config in AT.

**Pull request merged**